src/webots/nodes/WbIndexedLineSet.cpp

```diff
@@ -110,6 +110,13 @@ bool WbIndexedLineSet::sanitizeFields() {
     return false;
   }
 
+  const WbShape *const shape = dynamic_cast<WbShape *>(parentNode());
+  if (shape && shape->pbrAppearance()) {
+    parsingWarn(tr(
+      "The 'IndexedLineSet' node doesn't support 'PBRAppearance' in its 'appearance' field, please use 'Appearance' instead."));
+    return false;
+  }
```

Comment:
This test is now pointless right?

Reply:
No, it is still useful when loading a world file with a PBRAppearance for an IndexedLineSet.

Reply:
In that case this warning is not raised, because the PBRAppearance node is never instantiated because it does not respect the insertion rules:
```
WARNING: DEF INDEXED_LINE_SET Transform > Shape: Skipped unexpected PBRAppearance node in 'appearance' field of Shape node.
``